files/deploy-service.yaml

```diff
@@ -0,0 +1,88 @@
+---
+- name: Deploy packit-service into OpenShift
+  hosts: all
+  tasks:
+  - name: Download the deployment
+    git:
+        repo: 'https://github.com/packit-service/deployment.git'
```

Comment:
It's better to put that project as a `required-project` instead, then zuul will take care of copying the source.

Reply:
Nice

Reply:
But in this case do I have to setup zuul on this repo also? https://github.com/packit-service/deployment/

Reply:
You don't have to setup zuul for the deployment project (though you might want to gate it once there is a test for it), but we need to tell zuul about it, e.g. with https://softwarefactory-project.io/r/16110

When the Zuul application is installed on a project, you don't have to setup zuul.yaml in it, instead you can also configure the pipelines from the packit-service-zuul project.

Reply:
Ok got it, using which variable I can then access the project in other playbooks? 

E.g, when I add:
```
required-project: packit-service/deployment
```

Where it is going to be cloned?

Reply:
The project will be added the zuul.projects structure (you can see the variables here in the [zuul-info/inventory.yaml](https://softwarefactory-project.io/logs/88/88/7fe4b2bd187c30fce0fcef20ce110c4827f03e1e/check/packit-service-pre-commit/3113117/zuul-info/inventory.yaml) .
So that would be `{{ ansible_user_dir }}/{{ zuul.projects['github.com/packit-service/deployment'].src_dir }}`